packages/components/src/Components/Maintenance/Maintenance.js

```diff
@@ -35,13 +35,15 @@ const Maintenance = ({ utcStartTime, utcEndTime, startTime, endTime, timeZone, d
 };
 
 Maintenance.propTypes = {
-    utcStartTime: propTypes.string,
-    utcEndTime: propTypes.string,
-    startTime: propTypes.string,
-    endTime: propTypes.string,
-    timeZone: propTypes.string,
-    description: propTypes.node,
-    redirectLink: propTypes.string
+    utcStartTime: PropTypes.string,
```

Comment:
Maybe when we are here, we could make these `node`s

Reply:
Yeah but I think we could do a larger scale revision of prop types. There is a lot of this stuff all over the repository.